chainer_chemistry/datasets/molnet/molnet.py

```diff
@@ -0,0 +1,176 @@
+from logging import getLogger
+import os
+import shutil
+
+import numpy as np
```

Comment:
We are following chainer's coding guide that we won't use this abbreviation in the library code.
Please use `numpy`.